src/key-queue.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * The KeyQueue in Tram-One is a basic list of keys
+ * that needs to be persisted in the globalSpace.
+ *
+ * Currently this is used with useStore to keep track of what
+ * stores need to be associated with generated elements
+ */
```

Comment:
This KeyQueue (while different functionally) is similar to the EffectQueue. It is used to determine what store keys are generated during element generation, and tie it back to the element. 

The name `KeyQueue` is intentionally generic, as (hopefully) we may consolidate the EffectQueue into this, and the EffectStore into the KeyStore.